src/styles/main.scss

```
@@ -1,3 +1,204 @@
+$main-color: #060b35;
+$secondary-color: #616070;
+$accent-color: #00acdc;
+$black-color: #000;
+$white-color: #fff;
+$dirty-white: #f3f3f3;
+@import url("https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap");
+
 body {
   margin: 0;
+  font-family: Roboto, Helvetica, sans-serif;
+  font-size: 12px;
+  line-height: 14.6px;
+}
+
+.header {
+  display: flex;
+  height: 60px;
+  justify-content: space-between;
+  padding: 0 50px;
+}
+
+.logo {
+  width: 40px;
+  height: 40px;
+  align-self: center;
+}
+
+.nav__list {
+  padding: 0;
+  margin: 0;
+  font-family: "Roboto", sans-serif;
+  height: 100%;
+  display: flex;
+  list-style: none;
+}
+
+.nav__item {
+  font-size: 12px;
+  padding-left: 20px;
```

```suggestion
  margin-left: 20px;
```
Це краще робити марджинами